src/app/tasks/checker-inbox-and-tasks-tabs/client-approval/client-approval.component.html

```
@@ -8,21 +8,23 @@
-            matInput
-            placeholder="{{ 'labels.text.Filter by name' | translate }}"
-            (keyup)="applyFilter($event.target.value)"
-          />
-        </mat-form-field>
-      </div>
-      <div class="flex-60">
-        <button mat-raised-button color="success" *mifosxHasPermission="'ACTIVATE_CLIENT'" (click)="approveClients()">
-          <fa-icon icon="check" class="m-r-10"></fa-icon>{{ 'labels.buttons.Approve' | translate }}
-        </button>
-      </div>
+    <div style="display: flex; align-items: center; gap: 8px; margin-bottom: 16px">
```

HTMLHint (1.8.0)

[warning] Inline style [  style="display: flex; align-items: center; gap: 8px; margin-bottom: 16px" ] cannot be used.

(inline-style-disabled)